slasher/db/iface/interface.go

```diff
@@ -31,6 +31,9 @@ type ReadOnlyDatabase interface {
 	LatestIndexedAttestationsTargetEpoch(ctx context.Context) (uint64, error)
 
 	// MinMaxSpan related methods.
+	EpochSpans(ctx context.Context, epoch uint64) ([]byte, error)
+	SetValidatorSpan(ctx context.Context, spans []byte, validatorIdx uint64, newSpan detectionTypes.Span) ([]byte, error)
+	GetValidatorSpan(ctx context.Context, spans []byte, validatorIdx uint64) (detectionTypes.Span, error)
```

Comment:
It would be great if you could make a new `EpochSpans` interface of a []byte, and implement these functions, with the bytes being the receiver 😄

Reply:
ok, trying it

Reply:
great suggestion. ptal